playbooks/roles/cluster/tasks/cluster_create_generate_inventory.yml

```diff
@@ -59,6 +59,13 @@
   set_fact:
     domain: "{% if cluster_type is undefined %}{{ dev_domain }}{% elif cluster_type=='POC' %}{{ poc_domain }}{% endif %}"
 
+- name: Overwrite DNS task file if neccessary
+  get_url:
+    url: https://raw.githubusercontent.com/StevenTobin/openshift-tools/integreatly_cluster_create_3scale/ansible/roles/oa_aws_provisioning_dns_hook/tasks/main.yml
```

Comment:
Does the fact that this URL contains the StevenTobin username matter? Is there somewhere else we could host this file from?

Reply:
No, we can host it from anywhere it can pull the file from.